rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -78,7 +78,7 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
     implicit val costAccounting =
       CostAccountingAlg.unsafe[Task](CostAccount(Integer.MAX_VALUE))
     val result = withTestSpace { space =>
-      val reducer      = RholangOnlyDispatcher.create[Task, Task.Par](space).reducer
+      val reducer      = RholangOnlyDispatcher.create[Task, Task.Par](space)._2
```

Comment:
Our tests seem to have a very nasty habit of duplicating test setup for all test cases...

Would that be too much of an ask if I asked you to make this test define the dispatcher once? I'm fine if it's using a `def` :)

I think it'd be best if we changed `withTestSpace { space =>` to `withTestSpaceAndReducer { space, reducer =>`

Reply:
In few tests we use different reducer (`RholangAndScalaDispatcher`).

Reply:
I'm fine if you leave them out, and even more fine if you provide a separate method for them.